kylin-spark-project/kylin-spark-query/src/main/scala/org/apache/kylin/query/runtime/plans/AggregatePlan.scala

```diff
@@ -115,7 +137,7 @@ object AggregatePlan extends LogEx {
         val aggName = SchemaProcessor.replaceToAggravateSchemaName(index, funcName, hash, argNames: _*)
         if (funcName == FunctionDesc.FUNC_COUNT_DISTINCT) {
```

Comment:
should we replace "funcName == FunctionDesc.FUNC_COUNT_DISTINCT" with call.getFunc.isCountDistinct  ?

Reply:
Good suggestion. Done